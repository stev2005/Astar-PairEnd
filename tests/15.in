GTTACTCTGTTCGGACTGGTTTCGCCCAGGCCCACGTGCCCC
15
TTGAGAGGTGTCTAGATACAGCCAAT
CATTTGCAACGGAACCAGAACA
C
GG
GCCTCAGAGTA
ATCATG
TACTCGAGA
ATACATGACTGATCAGGAGAGCCTCG
CCGACCATGACGCACGAGGGATAACGCAT
GACAAGCCCTCTGGTTATCTGCGGCTAT
TTCAACAGTCAGGTACGTCGAAA
GGTCTGCGCGGCATTTATCTGGACC
TGGTTACGGAATTCCCGCAGA
CGAAAACCCATTTTAAC
CG
