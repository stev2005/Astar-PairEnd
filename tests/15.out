13
12
0
0
4
2
3
11
14
14
11
11
9
9
0
